playbook/app/pb_kits/playbook/pb_textarea/index.tsx

```diff
@@ -1,6 +1,8 @@
 import PbEnhancedElement from '../pb_enhanced_element'
 
 export default class PbTextarea extends PbEnhancedElement {
+  style: any
```

Comment:
Good job on the conversion! Just a few points: we try to stay away from the 'any' type as much as possible, did you try different types here and still get errors? We can always pair and try and figure it out together if you want!